internal/tools/kops/sizes.go

```diff
@@ -0,0 +1,41 @@
+package kops
+
+import "fmt"
+
+// ClusterSize is sizing configuration used by kops at cluster creation.
+type ClusterSize struct {
+	NodeCount  string
+	NodeSize   string
+	MasterSize string
+}
+
+// validSizes is a mapping of a size keyword to kops cluster configuration.
+var validSizes = map[string]ClusterSize{
+	"SizeAlef500":  sizeAlef500,
+	"SizeAlef1000": sizeAlef1000,
+}
+
+// sizeAlef500 is a cluster sized for 500 users.
+var sizeAlef500 = ClusterSize{
+	NodeCount:  "2",
+	NodeSize:   "t2.medium",
+	MasterSize: "m3.medium",
+}
+
+// sizeAlef1000 is a cluster sized for 1000 users.
+var sizeAlef1000 = ClusterSize{
+	NodeCount:  "4",
+	NodeSize:   "t2.medium",
+	MasterSize: "m4.large",
+}
```

Comment:
I know these are /super/ ballpark-y but these sizes seem a little small to be running a whole cluster installation (MM, DB, minio, elasticsearch, metrics, nginx). Probably fine for now but my guess is we'll need to bump them up later

Reply:
yeah, this is basically in line with the standard kops sizing if no overrides are given. These will need a bunch of tweaking so I figured it would be good to keep them smaller right now to combat AWS spend.